pkg/queue/cohort.go

```diff
@@ -35,3 +35,14 @@ func newCohort(name string) *cohort {
 func (c *cohort) GetName() string {
 	return c.Name
 }
+
+func (c *cohort) CCParent() hierarchy.CycleCheckable {
```

Comment:
nit: Is this used? Don't see it used on the diff.

Reply:
Yes, it satisfies the interface so we can use the cycle-checker. Added a comment.